CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 4.1.6
```

Comment:
Let's do a minor update and explicitly call out the requirement for logstash version